umm-spec-lib/test/cmr/umm_spec/test/umm_json.clj

```diff
@@ -89,6 +89,13 @@
         (util/update-in-all [:DataCenters :ContactPersons :ContactInformation :RelatedUrls] remove-nils)
         (util/update-in-all [:RelatedUrls] remove-nils))))
 
+(defn- remove-nil-related-urls
```

Comment:
This looks like it just removes _all_ related urls. Maybe update the description/name?

Reply:
updated the name to remove-related-urls and updated the description.